docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -488,10 +433,42 @@ class Sidebar extends React.Component<any, any> {
           />
           <Text color="white" content={pkg.version} size="medium" styles={logoStyles} />
         </Segment>
+        <Flex column>
+          <a
+            href={constants.repoURL}
+            target="_blank"
+            rel="noopener noreferrer"
+            style={topItemTheme}
+          >
+            <Box>
+              GitHub
+              <Icon name="github" styles={{ float: 'right' }} />
+            </Box>
+          </a>
+          <a href={changeLogUrl} target="_blank" rel="noopener noreferrer" style={topItemTheme}>
+            <Box>
+              CHANGELOG
+              <Icon name="file alternate outline" styles={{ float: 'right' }} />
+            </Box>
+          </a>
+          <Input
+            styles={topItemTheme}
+            fluid
+            clearable
+            icon={{ name: 'search', style: { padding: topItemTheme.padding } }}
+            placeholder="Search"
+            iconPosition="start"
+            role="search"
+            onChange={this.handleQueryChange}
+            value={this.state.query}
+            inputRef={this.searchInputRef}
+          />
+        </Flex>
         <Tree
-          defaultActiveIndex={activeCategoryIndex}
           items={allSections}
           renderItemTitle={titleRenderer}
+          activeIndex={this.state.activeCategoryIndex}
+          onActiveIndexChange={this.treeActiveIndexChanged}
```

Comment:
Let's add the method in the Tree in this PR, so we can test it with the real implementation. Then we can extract it to a separate PR

Reply:
Ok. Please take a look: https://github.com/stardust-ui/react/pull/1728